depmapomics/dm_omics.py

```diff
@@ -1025,7 +1030,7 @@ def assign_end_pos(
                     "encoding": "utf-8",
                 },
                 {
-                    "path": folder + "somatic_mutations_profile.maf.txt",
+                    "path": folder + "somatic_mutations_profile.maf.csv",
```

Comment:
Maf.csv suffix is not supported by IGV.

Reply:
This file should be reformatted to .txt in the subset & upload step downstream. Here I think we should keep it as a .csv so taiga can recognize it as a data frame, making it easier for subsetting

Reply:
I agree if taiga has such a format requirement.